tests/plugins/pastefromword/manual/shapesaretagged.md

```diff
@@ -6,6 +6,4 @@
 1. Copy its contents and paste to the editor.
 1. Click button `getHtml` below editor.
 
-**Expected:**
-  * For browsers with good `text/html` clipboard processing (Firefox, Chrome, Safari): First image tag had additional attribute `data-cke-is-shape="true"`. Second image is pasted without this attribute.
-  * For other browsers (Edge, IE, Mobile): There won't be `data-cke-is-shape="true"`. Browser will behave in an old way: display blank place or embed image by it's own.
+**Expected:** Single image is pasted to the editor. Shape present in word document is removed during paste process.
```

Comment:
You simplified it too much IMHO. It works as described in Chrome, FF, Safari. On IE11 both shape and image are pasted and on older IEs again only second image.

Additionally AFAIR or Safari and IEs < 11, the image is pasted, but it has invalid src, so basically the broken image is pasted (hard to tell if it is valid behaviour based on description).

TBH, I haven't thought about this earlier, but this test (`shapesaretagged`) was created to check if shapes are tagged with special attribute. Now shapes are removed so there is no purpose in keeping it and it could be simply removed, WDYT @msamsel? The case with removing shapes is already covered in other manual tests.

Sorry for the confusion, but I think it will be the best way, no sense in duplicating similar test.